api/candlepin-api-spec.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.0.2
+info:
+  title: Candlepin
+  description: Candlepin is a subscription management server written in Java. It helps with management of software subscriptions.
+  version: 3.0.0-draft
+
+servers:
+  - url: /candlepin
+
+security:
+  - basicAuth: []
+
+paths: {}
+
+components:
+  schemas:
+    ExceptionMessage:
+      description: An exception occurred
```

Comment:
Is this for a generic, unhandled exception? Do more specific exceptions get their own entry in this area?

Reply:
This essentially a port of `org.candlepin.common.exceptions.ExceptionMessage` to the swagger YAML. As resources require it we could have additional schemas (e.g. `org.candlepin.common.exceptions.DeletedConsumerExceptionMessage`). That looks like the only other place we've done this.

Reply:
That sounds fine, and we can discuss specifics if it comes up during actual implementation.